Sources/Apollo/MultipartResponseParsingInterceptor.swift

```diff
@@ -111,44 +110,37 @@ public struct MultipartResponseParsingInterceptor: ApolloInterceptor {
               response: response,
               completion: completion
             )
-          }
 
-          if let done = object["done"] as? Bool, done {
-            // Exit at this point because the router will close the connection; errors would have
-            // been reported or the subscription is complete.
+            // These are fatal-level transport errors, don't process anything else.
             return
           }
 
-          guard
-            let payload = object["payload"] as? JSONObject,
-            let data: Data = try? JSONSerializationFormat.serialize(value: payload)
-          else {
-            chain.handleErrorAsync(
-              MultipartResponseParsingError.cannotParsePayloadData,
-              request: request,
-              response: response,
-              completion: completion
+          if let payload = object["payload"] as? JSONObject {
```

Comment:
It looks here like if there is no `payload` that we don't throw an error or proceed onto the next part of the chain. This seems wrong to me. Is that behavior correct?